app/views/smart_answers/_debug.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if Rails.env.development? && params[:debug] %>
```

Comment:
Could we do this in a presenter/controller instead? I'd rather not be poking into Rails' internal state in a view.